client/src/utils/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './debug'
+export * from './helpers'
```

Comment:
I'm not a big fan of having a index file just to import everything. I think it's perfectly fine to `import { openURL } from '../utils/helpers'` and `import { debugDraw } from '../utils/debug'` because we don't have a ton of things that we are trying to import from many different files. So I suggest we remove this index.ts

Reply:
You're right, it's not necessary here. I already do it out of habit 😅